scripts/nn/test/grad_check.dml

```diff
@@ -949,6 +951,48 @@ cross_entropy_loss = function() {
   }
 }
 
+cross_entropy_loss2d = function() {
+  /*
+   * Gradient check for the cross-entropy loss function.
+   */
+  print("Grad checking the cross-entropy loss2d function.")
+
+  # Generate data
+  N = 3 # num examples
+  C = 10 # num targets
```

Comment:
small: can you add an extra space to each of these lines so that there are two spaces before the `#` instead of one?  Python style requires two spaces at the end of the line before any inline comments.